src/main.js

```diff
@@ -11,33 +11,34 @@ var homeBtn = document.querySelector('.home-button');
 var saveCoverBtn = document.querySelector('.save-cover-button');
 var viewSavedCoverBtn = document.querySelector('.view-saved-button');
 var savedCoverView = document.querySelector('.saved-view');
-
-
-
+///////////////////VARIABLE FOR INPUT FIELDS////////////////////
+var userCover = document.querySelector('#cover');
+var userTitle = document.querySelector('#title');
+var userDescript1 = document.querySelector('#descriptor1');
+var userDescript2 = document.querySelector('#descriptor2');
+var createCoverBtn = document.querySelector('.create-new-book-button');
 
 /////////// GLOBAL VARIABLES //////////
 var savedCovers = [
-  new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
```

Comment:
ohhh i think we gotta put this path back, seems like we forgot to use command +z

Reply:
okey dokey!